### DISCUSSION TITLE
绳网jb头像怎么换(╥_╥)

### DISCUSSION THREAD

**Discussion by TTGFCCt** [share121/inter-knot#419] [其他]
![Screenshot_20240729_170222_com viva pwdmanager p2](https://github.com/user-attachments/assets/d2030e31-8f0d-433c-be78-82481207c7f3)
不是，我搞了一个小时了这头像都换不了(╥_╥)求大佬指点

**Comment by Kamisato100** [comment]
在GitHub里面右上角点进去，点Setting，点Profile picture换自己的图片上去，最好不要太大

**Comment by duoduo13123678** [comment]
你需要在github换

**Comment by Xmug** [comment]
你要换Github的头像，因为这个绳网的原理就是读取Gihub项目的discussions区的公开信息，这也是为什么无论是发表主题贴还是回复都要跳转到Github才行。

**Comment by BSNHD** [comment]
直接在绳网主页面点左上角你的头像，跳转到github后再点头像再次跳转后点头像选upload那个就可以换了

**Comment by joke2-54** [comment]
回答：该过程总共有三个步骤。
           第一    点击你的绳网头像，等待跳转网页
           第二    点击跳转网页后的头像图片，再次等待跳转
           第三    点击右侧头像图片的左下方处的铅笔，点开后移动鼠标点击Upload a photo

**Comment by ConsistDust3838** [comment]
我也传不了图片，悲伤

**Comment by JonathanQZW** [comment]
摸摸头

**Comment by RHHero** [comment]
要换你的GitHub账号的头像哦，那边的头像会实时显示在绳网的👍
